pkgs/tools/filesystems/apfs-fuse/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub, fuse3, bzip2, zlib, attr, cmake }:
+
+stdenv.mkDerivation rec {
+  name = "apfs-fuse-${version}";
+  version = "20190723"; # No release tags yet, so just using date of last commit
+
+  src = fetchFromGitHub {
+    owner = "sgan81";
+    repo = "apfs-fuse";
```

Comment:
A common pattern here is to use `repo = pname;`.

Reply:
Left it as-is, because I changed `pname` to "apfs-fuse-unstable".